#! /usr/bin/env python

PACKAGE='theora_image_transport'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("post_processing_level", int_t, 0, "Post-processing level. Higher values can improve the appearance of the decoded images at the cost of more CPU.", 0, 0, 7)

exit(gen.generate(PACKAGE, "TheoraSubscriber", "TheoraSubscriber"))
